app/src/main/java/com/teamagam/gimelgimel/app/GGApplication.java

```diff
@@ -25,6 +26,10 @@ public void onCreate() {
         super.onCreate();
 
         CheckIfAppUpdated();
+
+        // Initialize the fresco plugin.
+        // Should be here instead of each activity
```

Comment:
Why not?
We want to initialize the library for the app, and not for specific activity